src/mouse-touchpad.vala

```diff
@@ -95,7 +95,17 @@ namespace MouseTouchpad {
 
         /* 'search' returns results like ("Keyboard → Behavior → Duration", "keyboard<sep>behavior") */
         public override async Gee.TreeMap<string, string> search (string search) {
-            return new Gee.TreeMap<string, string> (null, null);
+          var search_results = new Gee.TreeMap<string, string> ((GLib.CompareDataFunc<string>)strcmp, (Gee.EqualDataFunc<string>)str_equal);
```

Comment:
Looks like this whole section has 2 space indentation instead of 4

Reply:
Damn, I'll go over my other PR's